lib/broadside/deploy/ecs_deploy.rb

```diff
@@ -6,33 +7,58 @@
 
 module Broadside
   class EcsDeploy < Deploy
+
+    DEFAULT_DESIRED_COUNT = 0
```

Comment:
wait you mean in `EcsConfig`?  the constants are referenced here but that sort of makes sense...  not sure it's easier to read if it's in another file